inc/namespace.php

```diff
@@ -65,13 +68,28 @@ function load_redirects() {
 	require_once Altis\ROOT_DIR . '/vendor/humanmade/hm-redirects/hm-redirects.php';
 }
 
+/**
+ * Load Yoast SEO.
+ */
+function load_wpseo() {
+	$wpseo_file = Altis\ROOT_DIR . '/vendor/yoast/wordpress-seo/wp-seo.php';
+
+	// Define a fake WP SEO Premium File value if we don't have WP SEO Premium installed. This hides some of the upsell UI.
+	if ( ! class_exists( 'WPSEO_Premium' ) ) {
+		if ( ! defined( 'WP_CLI' ) || ! WP_CLI ) {
+			define( 'WPSEO_PREMIUM_FILE', $wpseo_file );
+			define( 'WPSEO_PREMIUM_VERSION', 8 );
```

Comment:
Think this was referenced in another (I guess based off this branch) - is the version important? Will we need to update it ever?

Reply:
As noted in the other ticket, it's not explicitly important in itself, just that it exists. (In testing it seemed like there were some behavioral differences if the value is numeric vs a boolean, hence making it an integer here.) But I've updated this value to use `Altis\get_version()` so it's dynamically reflecting the current Altis version rather than being hard-coded.

Reply:
Ah I didn't click that the `8` was in reference to the Altis version. If it doesn't mater what the actual value is then this is fine.